src/main/java/minicraft/level/tile/Tiles.java

```diff
@@ -31,8 +33,16 @@ public static void initTileList() {
 		tiles.put((short)17, new LavaTile("Lava"));
 
 		tiles.put((short)7, new RockTile("Rock"));
-		tiles.put((short)8, new TreeTile("Tree"));
-		tiles.put((short)9, new SaplingTile("Tree Sapling", Tiles.get("Grass"), Tiles.get("Tree")));
+		tiles.put((short)8, new TreeTile(TreeTile.TreeType.OAK));
+		// These are out of order because of the multiple tree types.
+		tiles.put((short)47, new TreeTile(TreeTile.TreeType.SPRUCE));
+		tiles.put((short)48, new TreeTile(TreeTile.TreeType.BIRCH));
+		tiles.put((short)49, new TreeTile(TreeTile.TreeType.ASH));
+		tiles.put((short)50, new TreeTile(TreeTile.TreeType.ASPEN));
+		tiles.put((short)51, new TreeTile(TreeTile.TreeType.FIR));
+		tiles.put((short)52, new TreeTile(TreeTile.TreeType.WILLOW));
```

Comment:
Pretty original selection.

I don't see the point of having so many tree sorts, though.

Reply:
For fancy visual effects.

Reply:
There are better ways of doing that.

Reply:
Using data?

Reply:
Damage is also used for data.

Reply:
No, rendering.

Reply:
If only depending on rendering, that would be quite random.

Reply:
Also, the sorts of tree is not really a lot, for bigger worlds unless you want to reduce the types of tree generated for smaller worlds.